src/Tizen.Applications.EventManager/Tizen.Applications/EventManager.cs

```diff
@@ -0,0 +1,100 @@
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void PublishAppEvent(string eventName, Bundle eventData)
+        {
+            Interop.AppEvent.ErrorCode err = Interop.AppEvent.EventPublishAppEvent(eventName, eventData.SafeBundleHandle);
+            if (err != Interop.AppEvent.ErrorCode.None)
+            {
+                ErrorFactory.ThrowException(err, "Publish event");
+            }
+        }
+
+        /// <summary>
+        /// Sends the User-Event to trusted receiver-applications.
+        /// </summary>
+        /// <param name="eventName">The event's name to send.</param>
+        /// <param name="eventData"> The event's data to send.</param>
+        /// <remarks>The application which has same certification with sender can receive the event.
+        /// The format of User-Event's name MUST be "event.{sender's appid}.{user-defined name}", refer to 'The name-format of User-Event' section,
+        /// If the event_name is invalid, throw InvalidOperationException</remarks>
+        /// <exception cref="ArgumentException">Thrown in case of an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void PublishTrustedAppEvent(string eventName, Bundle eventData)
+        {
+            Interop.AppEvent.ErrorCode err = Interop.AppEvent.EventPublishTrustedAppEvent(eventName, eventData.SafeBundleHandle);
+            if (err != Interop.AppEvent.ErrorCode.None)
+            {
+                ErrorFactory.ThrowException(err, "Publish trusted event");
+            }
+        }
+
+        /// <summary>
+        /// Keeps last User-Event data for receiver applications.
+        /// </summary>
+        /// <param name="eventName">The event's name to send.</param>
+        /// <remarks>The receiver applications will receive this last event data after adding their new handlers via ApplicationEventReceiver since the sender application called this method.
+        /// If a sender application sends same event via trusted method and non-trusted method, then a trusted receiver will get latest data regardless of trusted or non-trusted,
+        /// but non-trusted receiver will get the last data only from non-trusted method. The effect of this API continues during runtime.
+        /// That means when the sender application process restarts, the sender application needs to call this api again to make the event to keep the last event.</remarks>
+        /// <exception cref="ArgumentException">Thrown in case of an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown in case of any internal error.</exception>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
Method which returns non-void value must have exactly one 'returns' section. On the other hand, why does this method returns a boolean at all? In C#, methods should end with success or throw an exception.

Reply:
Done